web_ui/server.py

```
@@ -1108,6 +1110,431 @@ def _debug_process_snapshot(session_id: str | None = None) -> dict[str, object]:
+            meta_data = resolved_meta
+            if _stringify_preview_value(meta_data.get("category")) == "BOOK":
+                current_poster = _stringify_preview_value(meta_data.get("poster"))
+                if not _is_http_url(current_poster):
+                    enriched_meta = dict(meta_data)
+                    enriched_meta["poster"] = _book_cover_from_meta(meta_data, session_id)
+                    meta_data = enriched_meta
+            preview = _extract_execution_preview(meta_data, execution_path)
+            preview["awaiting_input"] = bool(process_info.get("awaiting_input"))
+            return preview
+        except Exception:
+            pass
```

Ruff (0.15.21)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)